cypress/support/commands.js

```diff
@@ -0,0 +1,326 @@
+import 'cypress-file-upload';
```

Comment:
```suggestion
import '../../apps/frappe/cypress/support/commands.js'
```
@ankush can you try to extend commands from frappe somewhat like this instead of copying?